assets/js/components/KeyMetrics/ConversionReportingNotificationCTAWidget.js

```diff
@@ -35,9 +35,15 @@ import { CORE_USER } from '../../googlesitekit/datastore/user/constants';
 import whenActive from '../../util/when-active';
 import ConversionReportingDashboardSubtleNotification from './ConversionReportingDashboardSubtleNotification';
 import { MODULES_ANALYTICS_4 } from '../../modules/analytics-4/datastore/constants';
+import { CORE_SITE } from '../../googlesitekit/datastore/site/constants';
+import { CORE_UI } from '../../googlesitekit/datastore/ui/constants';
+import { KEY_METRICS_SELECTION_PANEL_OPENED_KEY } from './constants';
 
 function ConversionReportingNotificationCTAWidget( { Widget, WidgetNull } ) {
 	const [ isSaving, setIsSaving ] = useState( false );
+
+	const { setValue } = useDispatch( CORE_UI );
```

Comment:
Since we use this action only in `handleSelectMetricsClick` let's move it closer, to the above that callback